docs/docs/integrations/vectorstores/pathway.ipynb

```diff
@@ -230,53 +144,34 @@
+    "## Your own pipeline"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Parsing Documents\n",
-    "\n",
-    "The vectorization pipeline supports pluggable parsers. If not provided, defaults to `UTF-8` parser. You can find available parsers [here](https://github.com/pathwaycom/pathway/blob/main/python/pathway/xpacks/llm/parser.py).\n",
+    "### Running in production\n",
+    "To have your own Pathway data indexing pipeline check the Pathway's offer for [hosted pipelines](https://pathway.com/solutions/ai-pipelines). You can also run your own Pathway pipeline - for information on how to build the pipeline refer to [Pathway guide](https://pathway.com/developers/user-guide/llm-xpack/vectorstore_pipeline/). You can use [Langchain components](https://pathway.com/developers/user-guide/llm-xpack/vectorstore_pipeline/#langchain) Pathway for processing documents. \n",
     "\n",
-    "An example parser that can read PDFs, Word documents and other formats is provided with `parsers.ParseUnstructured`: "
+    "A production deployment will typically run the server in a separate process. We recommend to run the Pathway data indexing pipeline in a container-based deployment environment, such as Docker or Kubernetes. For more info see [Pathway's deployment guide](https://pathway.com/developers/user-guide/deployment/)."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "from pathway.xpacks.llm import parsers\n",
-    "\n",
-    "parser = parsers.ParseUnstructured()\n",
+    "### Processing documents\n",
     "\n",
-    "vector_server = PathwayVectorServer(\n",
-    "    *data_sources,\n",
-    "    embedder=embeddings_model,\n",
-    "    splitter=text_splitter,\n",
-    "    parser=parser,  # we set the parser in the Vector Server\n",
-    ")"
+    "The vectorization pipeline supports pluggable components for parsing, splitting and embedding documents. For embedding and splitting you can use [Langchain components](https://pathway.com/developers/user-guide/llm-xpack/vectorstore_pipeline/#langchain) or check [embedders](https://pathway.com/developers/api-docs/pathway-xpacks-llm/embedders) and [splitters](https://pathway.com/developers/api-docs/pathway-xpacks-llm/splitters) available in Pathway. If parser is not provided, it defaults to `UTF-8` parser. You can find available parsers [here](https://github.com/pathwaycom/pathway/blob/main/python/pathway/xpacks/llm/parser.py)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### Configuring the cache\n",
-    "The Pathway vectorizing pipeline comes with an embeddings cache:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "vector_server.run_server(..., with_cache=True)"
+    "The Pathway vectorizing pipeline comes with an embeddings cache."
```

Comment:
We can remove this - we point to docs on running the pipeline, no need to duplicate it here